haproxy/README.md

```diff
@@ -18,7 +18,19 @@ The Haproxy check is included in the [Datadog Agent][2] package, so you don't ne
 
 #### Prepare HAProxy
 
-##### Versions < 2
+##### Using Prometheus
+
+The check supports a newer implementation starting with HAProxy version 2 (enterprise version 1.9rc1) that is based on a Prometheus endpoint:
+
+1. Configure your `haproxy.conf` using the [official guide][16].
+
+2. [Enable](#configuration) the setting `use_prometheus` in `haproxy.d/conf.yaml`.
+
+3. [Restart HAProxy to enable the stats endpoint][3].
+
+Note: The prometheus endpoint can also be set in prior version using the [HAProxy exporter][17]
+
+##### Using the stats endpoint
 
 The Agent collects metrics via a stats endpoint:
```

Comment:
I think we should add a comment that this is not the recommended way to setup the integration anymore, but rather still present as a legacy option for existing users. Something like this:

```suggestion
Note: this configuration strategy is provided as a reference for legacy users. If you are setting up the integration for the first time, consider using the Prometheus-based strategy described in the previous section.

The Agent collects metrics via a stats endpoint:
```